Cod3rsGrowth.Teste/TestesUnitarios/TesteRepositorios.cs

```diff
@@ -0,0 +1,123 @@
+        atorRepositorio.Inserir(new() {Id = 30, Nome = "Tom Hanks", IdFilme = 10, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 31, Nome = "Robin Wright", IdFilme = 10, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 32, Nome = "Gary Sinise", IdFilme = 10, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 33, Nome = "John Travolta", IdFilme = 11, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 34, Nome = "Uma Thurman", IdFilme = 11, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 35, Nome = "Samuel L. Jackson", IdFilme = 11, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 36, Nome = "Christian Bale", IdFilme = 12, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 37, Nome = "Heath Ledger", IdFilme = 12, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        atorRepositorio.Inserir(new() {Id = 38, Nome = "Aaron Eckhart", IdFilme = 12, Premios = new List<string> { "SAG Awards", "Black Reel Awards"}});
+        var listaEsperada = TabelasSingleton.ObterInstanciaAtores;
+        var nome = listaEsperada[0].Nome;
+        var id = listaEsperada[0].Id;
+
+        var lista = atorRepositorio.ObterTodos();
+
+        Assert.NotEmpty(lista);
+        Assert.Equal(nome, lista[0].Nome);
+        Assert.Equal(id, lista[0].Id);
+    }
+
+    [Fact]
+    public void obter_todos_retorna_lista_de_todos_os_usuarios_do_repositorio_quando_lista_nao_nula()
+    {
+        usuarioRepositorio.Inserir(new() {Nome = "Gabriel Detofol", IdUsuario = 1, Plano = PlanoEnum.Premium, Senha = "123", NickName = "Detofol29"});
+        usuarioRepositorio.Inserir(new() {Nome = "Marcos Paulo", IdUsuario = 2, Plano = PlanoEnum.Nerd, Senha = "123", NickName = "SilvaMarcosPaulo"});
+        usuarioRepositorio.Inserir(new() {Nome = "Rubens", IdUsuario = 3, Plano = PlanoEnum.Kids, Senha = "123", NickName = "DarthRubens"});
+        usuarioRepositorio.Inserir(new() {Nome = "André", IdUsuario = 4, Plano = PlanoEnum.Free, Senha = "123", NickName = "AndrezinDoGrau"});
+        var listaEsperada = TabelasSingleton.ObterInstanciaUsuarios;
+        var nome = listaEsperada[0].Nome;
+        var id = listaEsperada[0].IdUsuario;
+        var senha = listaEsperada[0].Senha;
+        var nick = listaEsperada[0].NickName;
+
+        var lista = usuarioRepositorio.ObterTodos();
+
+        Assert.NotEmpty(lista);
+        Assert.Equal(nome, lista[0].Nome);
+        Assert.Equal(id, lista[0].IdUsuario);
+        Assert.Equal(senha, lista[0].Senha);
+        Assert.Equal(nick, lista[0].NickName);
```

Comment:
aqui vc tá verificando apenas o primeiro elemento da lista, não tem necessidade se for assim, pode fazer só a verificação que a lista não está vazia e a quantidade de itens